modin/logging/config.py

```diff
@@ -120,19 +120,31 @@ def configure_logging():
     pandas_version = pkg_resources.get_distribution("pandas").version
     num_physical_cores = str(psutil.cpu_count(logical=False))
     num_total_cores = str(psutil.cpu_count(logical=True))
-    svmem = psutil.virtual_memory()
     logger.info(f"Modin Version: {modin_version}")
     logger.info(f"Pandas Version: {pandas_version}")
     logger.info(f"Physical Cores: {num_physical_cores}")
     logger.info(f"Total Cores: {num_total_cores}")
-    logger.info(f"Memory Total: {bytes_int_to_str(svmem.total)}")
-    logger.info(f"Memory Available: {bytes_int_to_str(svmem.available)}")
-    logger.info(f"Memory Used: {bytes_int_to_str(svmem.used)}")
 
     if LogMode.get() != "enable_api_only":
         mem_sleep = LogMemoryInterval.get()
+        mem_logger = logging.getLogger("modin_memory.logger")
+        mem_log_filename = f".modin/logs/memory_{job_id}.log"
```

Comment:
Can we structure these logs so they are in a directory?

Reply:
You mean a directory on a per job basis? Like `.modin/logs/job_<uuid>/api.1` and `.modin/logs/job_<uuid>/memory.1`?

Reply:
@naren-ponder yes, except what is the `.1` component?